javacord-core/src/main/java/org/javacord/core/DiscordApiBuilderDelegateImpl.java

```diff
@@ -103,6 +106,11 @@
      */
     private volatile boolean waitForServersOnStartup = true;
 
+    /**
+     * The intents. Default are all intents except the privileged
+     */
+    private final Set<Intent> intent = new HashSet<>();
```

Comment:
```suggestion
    private final Set<Intent> intents = Arrays.stream(Intent.values())
                      .filter(intent -> !intent.isPrivileged())
                      .collect(Collectors.toCollection(HashSet::new));
```
(not written in IDE, might not be 100% correct)